internal/main.go

```diff
@@ -119,7 +119,7 @@ func IsTerminal(out io.Writer) bool {
 	if !ok {
```

Comment:
This function is not used anymore, please remove. Then the PR is good to go. Thanks for your patience.